Code/FindRGL.cmake

```diff
@@ -78,7 +78,12 @@ if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/DOWNLOAD_RGL)
     file(REMOVE ${DEST_SO_DIR}/${RGL_LINUX_ZIP_FILENAME})
 
     file(REMOVE ${CMAKE_CURRENT_SOURCE_DIR}/DOWNLOAD_RGL)
-endif()
+else ()
+    message("Omitting the RobotecGPULidar library download. Warning:
+             This may also happen due to interruption of previous project configurations. If you have any issues related
+             to the libRobotecGPULidar.so file please make sure that the DOWNLOAD_RGL file is not present under the Code
+             directory.")
```

Comment:
```suggestion
    message(WARNING "Omitting the RobotecGPULidar library download.
             You can see this message when using Clion multi-profile project reload; it is fine.
             This may also happen due to interruption of previous project configurations. If you have any issues related
             to the libRobotecGPULidar.so file please make sure that the DOWNLOAD_RGL file is not present under the Code
             directory.")
```